charts/nr-ebpf-agent/values.yaml

```diff
@@ -62,11 +99,11 @@ pushPeriod: "15"
 ebpfAgent:
   image:
     # -- eBPF agent image to be deployed.
-    repository: us-west1-docker.pkg.dev/pl-dev-infra/nr-ebpf-agent-lp/ebpf-agent
+    repository: docker.io/newrelic/newrelic-ebpf-agent
     # -- The pull policy is defaulted to IfNotPresent, which skips pulling an image if it already exists. If pullPolicy is defined without a specific value, it is also set to Always.
     pullPolicy: IfNotPresent
     # -- The tag of the eBPF agent image to be deployed.
-    tag: 0.0.5
+    tag: nr-ebpf-agent_0.0.5
```

Comment:
Will need to create new images for the agent & client, the changes you are adding for TLS/Sampling/etc. are not present in these existing image tags so any of the respective configurations from the Helm chart will not be picked up by the containers